tests/erc7432/grant-handler.test.ts

```diff
@@ -0,0 +1,162 @@
+import { assert, describe, test, clearStore, afterAll, beforeEach } from 'matchstick-as'
+import { generateRoleId } from '../../src/utils/helper'
+import { createNewRoleGrantedEvent } from '../mocks/events'
+import { handleRoleGranted } from '../../src/erc7432'
+import { Bytes } from '@graphprotocol/graph-ts'
+import { createMockAccount, createMockNft } from '../mocks/entities'
+
+const tokenId = '123'
+const address1 = '0x1111111111111111111111111111111111111111'
+const address2 = '0x2222222222222222222222222222222222222222'
+const address3 = '0x3333333333333333333333333333333333333333'
+
+const tokenAddress = address1
+const grantee = address2
+const data = '0x1234567890'
+const expirationDate = '99999'
+
+describe('ERC Role Granted', () => {
+  afterAll(() => {
+    clearStore()
+  })
+  describe('When entities does not exists', () => {
+    describe('When role does not exists', () => {
+      beforeEach(() => {
+        clearStore()
+      })
+      test('Should skip if NFT does not exist', () => {
+        const roleId = Bytes.fromUTF8('0xGrantRole').toHex()
+
+        const event = createNewRoleGrantedEvent(roleId, tokenId, tokenAddress, grantee, expirationDate, data, address1)
+
+        handleRoleGranted(event)
+      })
+      test('Should skip if grantor does not exist', () => {
+        createMockNft(tokenId, tokenAddress, address2)
+
+        const roleId = Bytes.fromUTF8('0xGrantRole').toHex()
+        const event = createNewRoleGrantedEvent(roleId, tokenId, tokenAddress, grantee, expirationDate, data, address3)
+
+        handleRoleGranted(event)
```

Comment:
We should put some kind of assert to make the test valid. Maybe you could check with: 

```suggestion
        assert.entityCount('Role', 0)
         assert.entityCount('Account', 0)
         
        handleRoleGranted(event)
        
        assert.entityCount('Role', 0)
        assert.entityCount('Account', 0)
```

To make sure that neither an Account or Role was created